src/App.tsx

```diff
@@ -1,35 +1,12 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import ResetStyles from './styles/ResetStyles';
```

Comment:
프로젝트 설정 단계에서 `alias path`를 사용하도록 설정해놓은 것을 확인했는데요!
이 부분도 `alias path`를 사용하도록 수정하는게 좋을 것 같습니다~
```suggestion
import ResetStyles from '@/styles/ResetStyles';
```

<br/>

사람이 코드를 작성하다보니, 요런 부분을 놓칠 수 있는데요!  
`프로젝트 전체에 걸쳐 동일한 컨벤션을 유지하려면 어떻게 해야할까?`
`요런 실수를 어떻게 막을 수 있을까?`
를 고민해보셔도 좋을 것 같아요 ㅎㅎ